library/network/src/lib/y2firewall/firewalld/api/zones.rb

```diff
@@ -69,19 +85,51 @@ def list_sources(zone, permanent: permanent?)
           string_command("--zone=#{zone}", "--list-sources", permanent: permanent).split(" ")
         end
 
+        # @param zone [String] The firewall zone
+        # @param permanent [Boolean] if true it adds the --permanent option the
```

Comment:
I would omit mentioning `--permanent`. What about something like `if true it modifies the "permanent" configuration`? (or something similar).